src/sensors/counter/DistanceMeterEncoder.java

```diff
@@ -19,6 +19,11 @@ public double getRate() {
         return counter.getCount();
     }
 
+    @Override
+    public void setCount(int count) {
```

Comment:
You even might be able to see in some encoder interfaces that adding a `set` does not fit the FIRST interfaces.